mutiny-core/src/nodemanager.rs

```diff
@@ -82,7 +82,7 @@ pub struct NodeStorage {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 pub struct NodeIndex {
     pub child_index: u32,
-    pub lsp: Option<String>,
+    pub lsp: Option<LspConfig>,
```

Comment:
This won't be backwards compatible, could just rename to `lsp_config` or write a custom parser. Should be fine to change to `lsp_config` as there hasn't been any reason to change your LSP as of yet (voltage is only option)